### ISSUE TITLE
[non-mixer] Hiding the main menu

### ISSUE THREAD

**Issue by unfa** [original-male/non#219]
In the pursuit to minimize wasted screen space, is there a way to hide the Non Mixer's main menu from it's window?

![spectacle t28190](https://user-images.githubusercontent.com/491704/27102656-49d935d8-5086-11e7-9b04-096677542cdf.png)

When setting up the mixer configuration, it's very useful, but for live performance or recording, it might be unneeded. Some hotkey would be great for toggling the menu's visibility.

**Comment by original-male** [comment]
You mean just the menubar? (including the project name)?

**Comment by unfa** [comment]
Yes. The project name could also be put into the Window titlebar (when the menu is hidden).
And I'd probably hide the window decoration anyway to save even more space.
